apel/ldap/query.py

```diff
@@ -120,8 +122,10 @@ def fetch_specint(site, host='lcg-bdii.egi.eu', port=2170):
                 continue
 
             for fk in fks:
-                if fk.startswith(GLUE_CE_UNIQUE_ID):
-                    name = fk.split('=')[1]
+                ft_str = fk.decode('utf-8')
+
+                if ft_str.startswith(GLUE_CE_UNIQUE_ID):
+                    name = ft_str.split('=')[1]
```

Comment:
```suggestion
                fk_str = fk.decode('utf-8')

                if fk_str.startswith(GLUE_CE_UNIQUE_ID):
                    name = fk_str.split('=')[1]
```

Looks like a typo in the variable name.